packages/spotlight/docs/index.md

```diff
@@ -291,43 +290,7 @@ provided, the first selector that successfully matches a node is used.
 + Default: `''`
 
 If the focus originates from another container, you can define which element in
-this container receives focus first.
-
-`preserveId`
-+ Type: [boolean]
-+ Default: `false`
-
-Whether the container will preserve the id when it unmounts.
-
-#### Properties
-
-`containerId`
-+ Type: [string]
-
-Specifies the container id. If the value is `null`, an id will be generated.
-
-`spotlightDisabled`
-+ Type: [boolean]
-+ Default: `false`
-
-When `true`, controls in the container cannot be navigated.
-
-`spotlightMuted`
-+ Type: [boolean]
-+ Default: `false`
-
-Whether or not the container is in muted mode. When in muted mode, Spottable controls within the container
-can still gain focus, however their `:focus` CSS styles will not be applied, giving them the appearance
-of not having focus. Muting a container is generally done to temporarily disable CSS changes and
-default `onFocus` and `onBlur` events without removing focus from the container itself - which would
-happen if you disabled the container using `spotlightDisabled`.
-
-`spotlightRestrict`
-+ Type: [string]
-+ Values: [`'none'`, `'self-first'`, or `'self-only'`]
-+ Default: `'none'`
-
-Restricts or prioritizes focus to the controls in the current container.
+this container receives focus first. 
```

Comment:
How about preserving `preserveId` and some of the properties like `spotlightRestrict`?

Reply:
I restore preserveId , containerId, spotlightRestrict